opacus/accountants/utils.py

```diff
@@ -34,28 +50,31 @@ def get_noise_multiplier(
     Returns:
         The noise level sigma to ensure privacy budget of (target_epsilon, target_delta)
     """
-    if accountant != "rdp":
-        # TODO: rewrite method to accept GDP
-        raise NotImplementedError(
-            "get_noise_multiplier is currently only supports RDP accountant"
-        )
-
     eps = float("inf")
     sigma_min = DEFAULT_SIGMA_MIN_BOUND
     sigma_max = DEFAULT_SIGMA_MAX_BOUND
-    accountant = create_accountant(mechanism=accountant)
```

Comment:
I believe this line should stay here, no? If it's here, it is called once only when `get_noise_multiplier` is called. 

When it is moved it under `def get_epsilon()` method, it will be called more than once. Am I missing the context here?